content/spin/v1/python-components.md

```diff
@@ -21,7 +21,7 @@ url = "https://github.com/fermyon/developer/blob/main/content/spin/v1/python-com
 - [Storing Data in SQLite](#storing-data-in-sqlite)
 - [AI Inferencing From Python Components](#ai-inferencing-from-python-components)
 
-With <a href="https://www.python.org/" target="_blank">Python</a> being a very popular language, Spin provides support for building components with Python; [using an experimental SDK](https://github.com/fermyon/spin-python-sdk). The development of the Python SDK is continually being worked on to improve user experience and also add new features. 
+With <a href="https://www.python.org/" target="_blank">Python</a> being a very popular language, Spin provides support for building components with Python; [using an experimental SDK](https://github.com/fermyon/spin-python-sdk/tree/old-sdk). The development of the Python SDK is continually being worked on to improve user experience and also add new features. 
```

Comment:
"older experimental"?

Reply:
How about something like

```
With <a href="https://www.python.org/" target="_blank">Python</a> being a very popular language, Spin provides support for building components with Python.  The [Spin 1.x Python SDK](https://github.com/fermyon/spin-python-sdk/tree/old-sdk) is still available but is no longer actively developed.  More recent versions of the SDK are not compatible with Spin 1.x.
```

Not sure if that would cause any issues in context (and phrasing could be improved for sure) but something like that would be more explicit about the Spin 1 situation.